app/services/game-overlay/index.ts

```diff
@@ -105,7 +105,13 @@ export class GameOverlayService extends PersistentStatefulService<GameOverlaySta
   }
 
   async initializeOverlay() {
-    overlay.start();
+    let crashHandlerLogPath = '';
+    if (process.env.NODE_ENV !== 'production' || !!process.env.SLOBS_PREVIEW) {
```

Comment:
So to be clear, this logging will only happen in dev and preview.  Is that what we want for now?

Reply:
Yes. It will give us information from preview users and not affect regular users too much.